lib/xain/helpers.ex

```diff
@@ -37,6 +38,18 @@ defmodule Xain.Helpers do
   def pop!([item | rest]) do
     {rest, item |> Enum.reverse}
   end
+
+  def ensure_valid_contents(contents, _) when is_binary(contents) or is_list(contents) do
+    contents
+  end
+  def ensure_valid_contents(contents, tag_name) when is_number(contents) or is_atom(contents) do
+    Logger.debug "#{tag_name} has been called as #{tag_name}(#{inspect(contents)}, ...), but the first argument supposed to be a binary"
+    to_string(contents)
+  end
+  def ensure_valid_contents(contents, tag_name) do
+    Logger.warn "#{tag_name} has been called as #{tag_name}(#{inspect(contents)}, ...), but the first argument supposed to be a binary"
+    inspect(contents)
```

Comment:
@smpallen99 I think, we could use `text` tag to call `to_string`.
I suppose that there is no case when somebody really want to pass anything except a binary or a list of binaries as content of a tag. Each such the case is involuntary, but it isn't so critical to raise exception. For example, I had got the error on integer values, b/c wrote [id, name] in wrong order for select.
Although I don't mind to replace `Logger.warn` with `ArgumentError.exception` for catch-all clause.